pennylane_lightning_kokkos/src/util/KokkosUtil.hpp

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 Xanadu Quantum Technologies Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+/**
+ * @file KokkosUtil.hpp
+ * Kokkos utility functions.
+ */
+
+#include <Kokkos_Core.hpp>
+#include <bits/stdc++.h>
```

Comment:
Do you mean why include `KokkosUtil.hpp`? I thought I would keep things separate at first, and move somewhere else as you see fit. I would keep only the streaming operator `<<` and move it to `Util.hpp`.

Reply:
I mean in this case `#include <bits/stdc++.h>`. This is a catch-all header for including all STL headers. I think we likely do not need 99% of these, so it may be better to just include the ones we intend to need for compiling the functions/classes included here.

Reply:
Since this file has been merged into Util.hpp, I'll call this resolved :)